redfish-core/src/error_messages.cpp

```diff
@@ -928,6 +928,40 @@ void propertyValueIncorrect(crow::Response& res, const std::string& arg1,
     addMessageToErrorJson(res.jsonValue, propertyValueIncorrect(arg1, arg2));
 }
 
+/**
+ * @internal
+ * @brief Formats PropertyValueResourceConflict message into JSON
+ *
+ * See header file for more information
+ * @endinternal
+ */
+nlohmann::json propertyValueResourceConflict(const std::string& arg1,
```

Comment:
Isn't this generated from some tool? 

https://github.com/openbmc/bmcweb/commit/81856681e1feabe267c0d1998182c9226106aa6d

Reply:
Looks like the error_messages.cpp|hpp are not auto-generated based on the [parse_registries.py](https://github.com/ibm-openbmc/bmcweb/blob/1020/scripts/parse_registries.py#L57).

I think we need to manually add the support in that files to use the `messages` abstraction layer that used to return as a redfish response. - Please correct me if I am wrong.

Even, I am not seeing any obvious changes in that commit that you pointed out for the auto generation.

But, I think I need to change the `MessageId` major and minor version from `1.12.1` to `1.10.0` since the `propertyValueResourceConflict` message registry was added in that particular version.